lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -161,50 +161,58 @@ export default function SubmitGradeForm({ student }) {
-        <ValidationMessage
-          message={validationMessage}
-          open={showValidationError}
-          onClose={() => {
-            // Sync up the state when the ValidationMessage is closed
-            setValidationError(false);
-          }}
-        />
-      )}
-      <label className="SubmitGradeForm__label" htmlFor={gradeId}>
-        Grade (Out of 10)
-      </label>
-      <span className="SubmitGradeForm__grade-wrapper">
-        <input
-          className={classNames('SubmitGradeForm__grade', {
-            'is-saved': gradeSaved,
-          })}
+    <>
+      <form className="SubmitGradeForm" autoComplete="off">
+        {validationMessage && (
+          <ValidationMessage
+            message={validationMessage}
+            open={showValidationError}
+            onClose={() => {
+              // Sync up the state when the ValidationMessage is closed
+              setValidationError(false);
+            }}
+          />
+        )}
+        <label className="SubmitGradeForm__label" htmlFor={gradeId}>
+          Grade (Out of 10)
+        </label>
+        <span className="SubmitGradeForm__grade-wrapper">
+          <input
+            className={classNames('SubmitGradeForm__grade', {
+              'is-saved': gradeSaved,
+            })}
+            disabled={disabled}
+            id={gradeId}
+            ref={inputRef}
+            onInput={handleKeyDown}
+            type="input"
+            // @ts-expect-error - `defaultValue` is missing from `<input>` prop types.
+            defaultValue={grade}
+            key={student ? student.LISResultSourcedId : null}
+          />
+          <Spinner
+            className={classNames('SubmitGradeForm__fetch-spinner', {
+              'is-active': gradeLoading,
+              'is-fade-away':
+                !gradeLoading && student && student.LISResultSourcedId,
+            })}
+          />
+        </span>
+        <button
+          type="submit"
+          className="SubmitGradeForm__submit"
           disabled={disabled}
-          id={gradeId}
-          ref={inputRef}
-          onInput={handleKeyDown}
-          type="input"
-          // @ts-expect-error - `defaultValue` is missing from `<input>` prop types.
-          defaultValue={grade}
-          key={student ? student.LISResultSourcedId : null}
-        />
-        <Spinner
-          className={classNames('SubmitGradeForm__fetch-spinner', {
-            'is-active': gradeLoading,
-            'is-fade-away':
-              !gradeLoading && student && student.LISResultSourcedId,
-          })}
-        />
-      </span>
-      <button
-        type="submit"
-        className="SubmitGradeForm__submit"
-        disabled={disabled}
-        onClick={onSubmitGrade}
-      >
-        <Icon classes="SubmitGradeForm__check-icon" name="check" /> Submit Grade
-      </button>
+          onClick={onSubmitGrade}
+        >
+          <Icon classes="SubmitGradeForm__check-icon" name="check" /> Submit
```

Comment:
This text wrapping is a bit awkward. Can we fix that without having Prettier revert it?

Reply:
Nope. Prettier seems pretty determined here. When next touching this component, I'd like to convert this over to a "real" Button component anyway.